robocup/game.json

```diff
@@ -0,0 +1,36 @@
+{
+  "type": "NORMAL",
+  "kickoff": "red",
+  "class": "KID",
+  "host": "127.0.1.1",
+  "minimum_real_time_factor": 0,
+  "side_left": "blue",
+  "supervisor": "test_supervisor",
```

Comment:
This supervisor line breaks the build. Removing it fixes it, it doesn't exist in the standard build of webots so I'm not sure what it's purpose is